src/main/tls.c

```diff
@@ -3377,7 +3377,19 @@ SSL_CTX *tls_init_ctx(fr_tls_server_conf_t *conf, int client)
 		 */
 		SSL_CTX_sess_set_cache_size(ctx, conf->session_cache_size);
 
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+		SSL_CTX_set_num_tickets(ctx, 1);
+#endif
 	} else {
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+		/*
+		 *	This controls the number of stateful or stateless tickets
+		 *	generated with TLS 1.3.  In OpenSSL 1.1.1 it's also
+		 *	required to disable sending session tickets,
+		 *	SSL_SESS_CACHE_OFF is not good enough.
+		 */
+		SSL_CTX_set_num_tickets(ctx, 0);
+#endif
```

Comment:
I've pulled this into v3.0.x.  It's needed for OpenSSL 1.1.1, even without TLS 1.3.